src/mocks/homepage/success.json

```diff
@@ -62,7 +62,7 @@
         "serviceDescriptionFollowUp": "Collecting participant specific events and outcomes by matching to existing data sets.",
         "signPostDescription1": "Need help finding appropriate data services, or any other part of the UK journey to getting your study started? The UK offers multiple services and teams of professionals who can support you.",
         "signPostDescription2": "If you provide services that support Find, Recruit or Follow-up activties and would be interested in inclusion within this website, we would love to hear from you.",
-        "metaTitle": "Homepage Title",
```

Comment:
Add this title back into the file, and run the unit tests. 
You can run the unit tests with the command line: npm run test